homeassistant/components/vicare/binary_sensor.py

```diff
@@ -112,6 +112,11 @@ class ViCareBinarySensorEntityDescription(
         device_class=BinarySensorDeviceClass.RUNNING,
         value_getter=lambda api: api.getOneTimeCharge(),
     ),
+    ViCareBinarySensorEntityDescription(
+        key="device_error",
+        device_class=BinarySensorDeviceClass.PROBLEM,
+        value_getter=lambda api: len(api.getDeviceErrors()) > 0,
```

Comment:
@zweckj `api.getDeviceErrors()` is giving a list of errors **codes** for the device. I chose this to be a binary sensor for simplicity. But would it also be acceptable to just display one error and have the error code as value and maintain the meaning of that in the translation files or is this not intended for such cases?

Reply:
Ok, then I leave it like this.